marshmallow/decorators.py

```diff
@@ -0,0 +1,67 @@
+PRE_DUMP = 'pre_dump'
+POST_DUMP = 'post_dump'
+PRE_LOAD = 'pre_load'
+POST_LOAD = 'post_load'
+
+
+def pre_dump(*args, **kwargs):
```

Comment:
For documentation purposes, it might be clearer to explicitly include `raw` in these signatures. The `tag_processor` includes the `raw` value in the tuple tags anyway.